### PR TITLE
feat(app): change Blazor Server App to Blazor WebAssembly App

### LINKED ISSUE #36: Change to Blazor WASM app
~~as PWA - change from current Blazor-Server-App~~ see #52

### PR DESCRIPTION
**Pull request by Flash0ver**
closes #36

### CODE REVIEW

File: source/production/F0.Minesweeper.App/F0.Minesweeper.App.csproj [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+
+	<PropertyGroup>
+		<TargetFramework>net5.0</TargetFramework>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="5.0.0" />
+		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="5.0.0" PrivateAssets="all" />
```
Comment (dev-in-disguise) [inline source/production/F0.Minesweeper.App/F0.Minesweeper.App.csproj:9]: Do we want to update to the latest stable release versions `5.0.2`?

**Review by dev-in-disguise — APPROVED** [review]
Lovely :) 
Although I noted a possible improvement with the upgrade of the two nugets I still approve.

File: source/production/F0.Minesweeper.App/F0.Minesweeper.App.csproj (hunk 1 above)
Comment (Flash0ver) [reply source/production/F0.Minesweeper.App/F0.Minesweeper.App.csproj]: I made it `5.0.3` - and also updated the .NET SDK (`global.json`)

Comment (Flash0ver) [reply source/production/F0.Minesweeper.App/F0.Minesweeper.App.csproj]: also added explicit [PreferExactMatches](https://docs.microsoft.com/en-us/aspnet/core/migration/31-to-50?tabs=visual-studio#changes-to-blazor-app-routing-logic-in-501-and-further-5x-releases-up-to-60)

**Pull request merged**